core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -564,6 +568,20 @@ def _fit(self,
         X = self.preprocess(X)
         self.model = self.model.fit(X, y)
 
+    def _apply_temperature_scaling(self, y_pred_proba):
```

Comment:
Move the `if self.temperature_scalar is not None` check to the predict_proba method. When `_apply_temperature_scaling` is called, it shouldn't need to check if temperature scaling is set or not. By calling it in the first place implies it has to be set.

Reply:
Ensure this is done for both AbstractModel and BaggedEnsembleModel